src/web/lib/getZIndex.test.tsx

```diff
@@ -0,0 +1,26 @@
+import { getZIndex } from './getZIndex';
+
+describe('getZIndex', () => {
+    it('gets the correct zindex for group and sibling', () => {
+        expect(
+            getZIndex({
+                group: 'bodyGroup',
```

Comment:
Can we mock these so we don't have to change tests each time we change? If someone has thoughts on how to do that, that'd be good.